Userland/Libraries/LibELF/Image.cpp

```diff
@@ -16,6 +16,16 @@
 #include <LibELF/Validation.h>
 #include <limits.h>
 
+// FIXME: This define comes from Kernel/API/POSIX/sys/types.h. Ideally, it would
+//        be shared between Kernel, Userland and Lagom without problems;
+//        however, because Lagom does not include Serenity's LibC/sys/types.h,
+//        we don't get PTHREAD_STACK_MAX (it's not defined in POSIX anyway).
+//        Figure out the right place for this to go so we don't have to inline
+//        the define.
+#ifndef PTHREAD_STACK_MAX
+#    define PTHREAD_STACK_MAX (32 * 1024 * 1024) // 32MiB
+#endif
+
```

Comment:
This is pretty bad and is currently a merge blocker (at least, I wouldn't merge it :^). Is there a better way to do this?

Reply:
Side point, but: eek, won't that include the `// 32MiB` comment verbatim wherever `PTHREAD_STACK_MAX` is used?

Reply:
Comments are stripped before preprocessor replacement happens. Otherwise this would fail with a syntax error already.

Reply:
Ah, I didn't know that. Sorry!